frontend/style.css

```diff
@@ -0,0 +1,124 @@
+body {
+    background-color: #303C49;
+    padding: 0%;
+    margin: 0%;
+    color: red;
+    font-family: Roboto;
+}
+
+/* middle buttons */
+
+.center-buttons {
+    height: 68.8%;
+    width: 100%;
+    display: flex;
+}
+
+.mechanic-buttons {
+    display: grid;
+    grid-template-columns: auto auto auto auto;
+    width: 80%;
+    gap: 4%;
+    margin: 2%;
+}
+
+.other-buttons {
+    display: grid;
+    grid-template-columns: auto;
+    width: 20%;
+    gap: 4%;
+    margin: 2%;
+}
+
+.middle-button {
+    border-radius: 999px;
+    background: white;
+    border: none;
+    color: #303C49;
+    font-size: 1.5vw;
+    font-weight: bold;
+}
+
+.middle-button:hover {
+    background-color: #bfbfbf;
+}
+
+
+/* bottom bar */
+
+.bottom-bar {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    height: 22%;
+    gap: 5%;
+}
+
+.file-chooser {
+    color: #303C49;
+    background-color: white;
+    border: none;
+    height: 75%;
+    width: 100%;
+    margin-left: 2%;
+    align-self: center;
+    font-size: 4vw;
+    font-weight: 600;
+}
+
+.file-chooser:hover {
+    background-color: #bfbfbf;
+}
```

Comment:
tohle je stejný jako .middle-button:hover, měl by to bejt jeden blok, kde ty selektory jsou odčárkovaný

```css
.file-chooser:hover, .middle-button:hover { ... }
```